spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -285,12 +289,14 @@ function sendFungibleTokens(
   sender: string,
   receiver: string,
   memo: string,
-  forwardingPath: []ForwardingInfo,
+  forwardingPath: ForwardingInfo,
   sourcePort: string,
   sourceChannel: string,
   timeoutHeight: Height,
   timeoutTimestamp: uint64, // in unix nanoseconds
 ): uint64 {
+  // memo and forwardingPath cannot both be non-empty
+  abortTransactionUnless(memo != "" && forwardingPath != nil)
```

Comment:
Actually, couldn't we get rid of one of of the two memos? If the memo supposed to trigger an action on the final destination of the tokens, why cannot we have something like this?

```
interface FungibleTokenPacketDataV2 {
  tokens: []Token
   sender: string
   receiver: string
   memo: string
   forwardingPath: ForwardingInfo
}

interface ForwardingInfo {
   hops: []Hop
}
```

But then it would not be possible to trigger any action on intermediary hops. Is that desired behaviour?